addons/zeus/config.cpp

```diff
@@ -41,12 +41,18 @@ class CfgPatches {
             QGVAR(moduleAddSpareWheel)
         };
     };
+    class GVAR(fastRoping): ADDON {
+        units[] = {
+            QGVAR(moduleAddOrRemoveFRIES)
+        };
+    };
 };
 
 class ACE_Curator {
     GVAR(captives) = "ace_captives";
     GVAR(medical) = "ace_medical";
     GVAR(cargoAndRepair)[] = {"ace_cargo", "ace_repair"};
+    GVAR(fastRoping) = "ace_fastroping";
```

Comment:
I still think this should be `GVAR(fastroping)` like the module, everywhere else as well, `fastRoping` capitalization looks weird IMO. `GVAR(cargoAndRepair)` is exception because it's for 2 components.